src/catalua.h

```diff
@@ -3,9 +3,90 @@
 #define CATALUA_H
 
 #include "int_id.h"
+#include "enums.h"
+#include "item.h"
 
 #include <string>
 #include <sstream>
+#include <vector>
+#include <list>
+
+/**
+ * Class that contains callback argument name and argument value (in different types).
+ */
+class CallbackArgument
+{
+    private:
+        std::string name;
+        std::string type;
+        int value_integer;
+        double value_double;
+        float value_float;
+        std::string value_string;
+        tripoint value_tripoint;
+        item value_item;
+    public:
+        CallbackArgument( std::string arg_name, int arg_value ) {
+            name = arg_name;
+            type = "integer";
+            value_integer = arg_value;
+        };
```

Comment:
Those semicolons after a function definition in a class definition are unnecessary and inconsistent (you have them only after constructors, but not after normal functions).